.github/workflows/e2e-upgrade.yaml

```diff
@@ -12,7 +9,16 @@ jobs:
       matrix:
         include:
           - test: TestV4ToV5ChainUpgrade
+            chain-image: ghcr.io/cosmos/ibc-go-simd
+            chain-a-tag: v4.0.0
+            chain-b-tag: v4.0.0
+            chain-upgrade-tag: pr-2144 # TODO: backport default upgrade handler to release/v5.x.x
```

Comment:
is there a pr for this?

Reply:
backporting the default upgrade handler here: #2551 #2552